OpenSim/Tools/CorrectionController.cpp

```diff
@@ -275,16 +280,22 @@ void CorrectionController::extendConnectToModel(Model& model)
             // the controller is removed, so are all the actuators it added.
             adoptSubcomponent(actuator);
             setNextSubcomponentInSystem(*actuator);
-        }
             
-        actuator->setOptimalForce(1.0);
+            std::cout << " CorrectionController::extendConnectToModel(): "
+                << name << " added " << std::endl;
+
+            actuator->setOptimalForce(1.0);
+        }
         
-        updActuators().adoptAndAppend(actuator);
+        // Add to the Controller's list of Actuators (no ownership).
+        addActuator(*actuator);
```

Comment:
Is it true that there's no more ownership or is the ownership now maintained through the subcomponents list? some clarifying comment would help

Reply:
`Controller::addActuator` does not take ownership and that was definitely not changed here. I added the comment for clarity. The `Model` generally owns the actuators being controlled. In this case, the `CorrectionController` owns the actuators it creates and includes them as its subcomponents, yes.